README.md

```diff
@@ -42,7 +42,7 @@ kubescape scan --submit --enable-host-scan --verbose
 
 </br>
 
-> Kubescape is an open source project. We welcome your feedback and ideas for improvement. We’re also aiming to collaborate with the Kubernetes community to help make the tests more robust and complete as Kubernetes develops.
+> Kubescape is an open-source project. We welcome your feedback and ideas for improvement. We’re also aiming to collaborate with the Kubernetes community to help make the tests more robust and complete as Kubernetes develops.
```

Comment:
thought: while grammatically correct, the hyphenated spelling is not universally adopted

This one is interesting, as the jury is still out on hyphenation.

On one hand, English grammar suggests we hyphenate compound adjective-like modifiers, so “open-source project” seems to be the grammatically correct way. The Economist style guide advocates for this rule, while the Associated Press style prefers dropping the hyphen. People also use the non-hyphenated version more in their searches, as evident by a Google Trends [comparison](https://trends.google.com/trends/explore?date=all&q=open-source,open%20source). And we want to be noticed. So actual user data favors the non-hyphenated spelling.

Moreover, Google Open Source communications style guide [advises](https://opensource.google/documentation/reference/comms/style-guide#quick_reference) to forgo hyphenation:
> Open source: Don’t capitalize or hyphenate open source unless it’s part of a proper noun (like Google Open Source).

So [does](https://developers.google.com/style/word-list?hl=en) the Google developer documentation style.

Therefore, in my opinion, we should stick to the grammatically wrong, but more widely adopted non-hyphenated spelling.

Reply:
@dwertent what do you think?

Maybe we should adopt a style guide—for example, Google’s—so we don’t waste time and energy arguing about style?

Reply:
ok I understand your point

Reply:
@vladklokun  hey can you help me I want to contribute to kubescape but am new to DevOps so can you help me to figure out how should I contribute  back to the community

Reply:
or if you you are interested in making style guide then I can help with you guys